packages/eslint-config-babel/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@integreat-app/eslint-config-babel",
+  "version": "0.2.1",
```

Comment:
I think the version should be 0.1.0 again.

Reply:
Or we leave it because it did just change the name.

Reply:
I would just leave the version number or maybe even increment it to 0.3.0 to avoid conflicts and signal the restructuring...